SPECS-EXTENDED/ocaml-ounit/ocaml-ounit.spec

```diff
@@ -23,6 +23,9 @@ Source0:        %{url}/releases/download/v%{version}/%{srcname}-v%{version}.tbz
 # compatibility package for older OCaml installations.  Patch it out instead.
 # Upstream does not want this patch until stdlib-shims is obsolete.
 Patch0:         %{name}-stdlib-shims.patch
+# Source: Fedora 35
+# https://src.fedoraproject.org/rpms/ocaml-ounit/blob/f35/f/ounit-v2.2.4-remove-Thread-kill.patch
+Patch1:         remove-thread-kill.patch
 
 BuildRequires:  ocaml >= 4.02.3
```

Comment:
Can we still build with lower versions or should we do `>= 4.13` here?

Reply:
We can still build with lower versions here, this just enables package building with more recent versions of ocaml.